components/NewPersonView/PersonHistory.tsx

```diff
@@ -0,0 +1,53 @@
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import Spinner from 'components/Spinner/Spinner';
+import React from 'react';
+import { Case } from 'types';
+import { useCasesByResident } from 'utils/api/cases';
+import { useUnfinishedSubmissions } from 'utils/api/submissions';
+import PersonTimeline from './PersonTimeline';
+
+interface Props {
+  personId: number;
+}
+
+const PersonHistory = ({ personId }: Props): React.ReactElement => {
+  const {
+    data: casesData,
+    size,
+    setSize,
+    error: casesError,
+    isValidating,
+  } = useCasesByResident(personId);
+  const { data: submissionsData } = useUnfinishedSubmissions(personId);
+
+  const events = [] as Case[];
+  for (let i = 0; casesData !== undefined && i < casesData.length; i++) {
+    events.push(...casesData[i].cases);
+  }
```

Comment:
Nit pick: would I be right in saying this `for` loop could go further down, underneath the `<ErrorMessage />` if statement?

Reply:
good point, save some CPU cycles :)